docs/components.md

```
@@ -159,22 +159,31 @@ This modal is based on [svelte-modals](https://svelte-modals.mattjennings.io/) w
 🌙
 ## MoonBase components
 
+Renderer: Takes JSON definitions and based on the type property shows a specific widget (e.g. input, checkbox, dropdown, ...)
+Widget: Complex, interactive components with editing capabilities
+
 <img width="30" src="https://github.com/user-attachments/assets/b0e8af99-ed76-422a-8bd1-bfbd9e0f4c44"/>
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)